scripts/ci/utils/generate-changelog

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
```

Comment:
We add the standard-version library, but won't use it to generate the changelog. It seems weird that we'd add the library, and still use this custom written script. Isn't the whole point of using standard version to offload the changelog generation to the library?

Reply:
yeah, I havent changes this yet, because i didnt want to change too much at once. we will/should change this script in the future to actually read the generated `CHANGELOG.md` file.

Reply:
Okay makes sense. Then we'll have two options:

1. Format the `standard-version` output into the version dcos expects PR descriptions to be (more manual scripting)
2. Ask dcos release people (like Senthil), if it's okay to have our PRs use the changelog format that `standard-version` outputs.

Reply:
3. keep some kind of script which fetches our `CHANGELOG.md` and extracts the information needed for a DC/OS PR.

But yeah, we should try to get 1. working, dima and orlando both say its possible 🤷‍♂️

Reply:
Yes, by 1) I actually was referring to your 3). And I agree that these are all three options 👍